### DISCUSSION TITLE
자동차

### DISCUSSION THREAD

**Discussion by sabalmyeon** [naverz/zepeto-studio-kor#508] [4. General]
유니티에서 제페토 빌드잇재료들을 가져와서 자동차를 꺼냈는데 어떻게 타나요? 탈 수 있는 방법있나요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 빌드잇 패키지에 포함된 자동차는 탈것이 아닌 꾸미기용 오브젝트입니다. 🚙 🚗 🚕
탈것을 구현하고 싶다면 유니티의 Vehicle 시스템 구현을 참고해서 직접 만들어야 합니다.
감사합니다.
